numpy/random/setup.py

```diff
@@ -52,78 +54,87 @@ def generate_libraries(ext, build_dir):
     PCG64_DEFS = []
     # One can force emulated 128-bit arithmetic if one wants.
     #PCG64_DEFS += [('PCG_FORCE_EMULATED_128BIT_MATH', '1')]
+    depends = ['__init__.pxd', 'c_distributions.pxd', '_bit_generator.pxd']
```

Comment:
`_bit_generator.pxd` existed in 1.18, but was not copied into the wheel. The use of `depends` on line 137 should fix that.